content_schemas/dist/formats/worldwide_organisation/frontend/schema.json

```diff
@@ -272,6 +272,14 @@
           "description": "Contact details for this Worldwide Organisation",
           "$ref": "#/definitions/frontend_links"
         },
+        "ordered_non_primary_role_people": {
```

Comment:
In terms of naming things this could be `ordered_other_role_people` to be consistent with Whitehall?

Reply:
But I think either is fine tbh

Reply:
The reason I chose the clunky "non-primary" is that in whitehall, "other roles" doesn't include secondary roles. When displaying, "secondary" and "other" get mushed together in the view, so they get away from having a name. I found it confusing when I was trying to figure out what happens to the secondary role person, so I chose to make that a little more accurate. (`secondary_and_other` would feel even clunkier 😅 )